.github/workflows/helm-release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release Helm Chart
+
+on:
+  workflow_dispatch:
+
+jobs:
+  helm-release:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    # if: ${{ github.repository_owner == 'cryostatio' }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+      - name: Get release version
+        id: release-version
+        run: |
+          echo "tag=v$(yq .version charts/cryostat/Chart.yaml)" >> $GITHUB_OUTPUT
+      - name: Get previous version
+        id: previous-version
+        uses: WyriHaximus/github-action-get-previous-tag@v1
+        with:
+          fallback: 0.1.0
```

Comment:
This action doesn't seem to consider the current branch/tag, so if for some reason we needed to release a 0.2.x after 0.3.0 has been released, it will return 0.3.0 as the tag to compare against. We don't plan on doing this now, but it might cause problems in the future.

Another problem is that while it accepts a pattern to match tags with, it uses it as a prefix only. So this could match things like `0.4.0-BETA` for example, and there's no way to avoid it.

We could avoid the second problem by calling `git for-each-ref` directly:
`git for-each-ref --sort=-creatordate --count 1 --format="%(refname:short)" "refs/tags/v[0-9]*.[0-9]*.[0-9]*"`.

Or we could handle both issues with something like this:
```bash
# Filter tags by regex, combine with the Chart version, sort by version number, and output the preceeding version.
version="$(yq .version charts/cryostat/Chart.yaml)"; \
{ echo v${version}; git tag -l | grep -E '^v[0-9]\.+[0-9]+\.[0-9]+$'; } | \
sort -V | grep -B1 "${version}" | head -n1
```
What do you think?

Reply:
Ahh understood! Sounds good! Updated just now thanks!

Reply:
Also, regex should instead be `^v[0-9]+\.[0-9]+\.[0-9]+$`?

Reply:
Ah sorry, typo.